### PR TITLE
Added tests for transactions for 500 errors caused by notifications

### PR DESCRIPTION
**Pull request by stevenbal**
issue https://github.com/open-zaak/open-zaak/issues/399

requires https://github.com/VNG-Realisatie/vng-api-common/pull/122 to be merged and vng_api_common version to be bumped

### CODE REVIEW

**Comment by joeribekker** [comment]
vng-api-common PR got merged, just needs a version bump

**Comment by stevenbal** [comment]
@sergei-maertens have you already added a new version of 1.0.x to pypi for vng_api_common?

**Comment by sergei-maertens** [comment]
@stevenbal just pushed 1.0.36 to PyPI :)

File: src/openzaak/components/zaken/tests/test_notifications_send.py [hunk 1]
```diff
@@ -627,3 +632,82 @@ def test_zaakbesluit_create_fail_send_notification_create_db_entry(self):
...
+            "startdatum": "2012-01-13",
+            "toelichting": "Een stel dronken toeristen speelt versterkte "
+            "muziek af vanuit een gehuurde boot.",
+            "zaakgeometrie": {
+                "type": "Point",
+                "coordinates": [4.910649523925713, 52.37240093589432],
+            },
+        }
+
+        # with self.assertRaises(ConnectionError):
```
Comment (stevenbal) [inline src/openzaak/components/zaken/tests/test_notifications_send.py]: @sergei-maertens maybe you know why, but for some reason this fails in travis. The exception does show up in the logs, but the assertion fails

**Comment by sergei-maertens** [comment]
Am looking into this :+1:

**Pull request merged**